File: Reverse_Words_in_a_String_III.java
class Solution {
    public String reverseWords(String s) {
        // Split the input string into words and reverse each word
        String[] words = s.split(" ");
        StringBuilder result = new StringBuilder();
        for (String word : words) {
            StringBuilder reversedWord = new StringBuilder(word).reverse();
            result.append(reversedWord).append(" ");
        }
        result.deleteCharAt(result.length() - 1);  // Remove the extra space at the end
        
        // Convert StringBuilder to String
        return result.toString();        
    }
}

File: Reverse_Words_in_a_String_III.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    // Split the input string into words and reverse each word
    const words = s.split(' ');
    const reversedWords = words.map(word => word.split('').reverse().join(''));
    
    // Join the reversed words to form the final result
    return reversedWords.join(' ');    
};
